projects/ion/src/lib/smart-table/smart-table.component.spec.ts

```diff
@@ -635,3 +637,68 @@ describe('Table without Data', () => {
     expect(screen.getByTestId('ion-checkbox')).toBeDisabled();
   });
 });
+
+describe('Table with cell events', () => {
+  const columnsWithCellEvent: Column[] = [
+    {
+      key: 'name',
+      label: 'Nome',
+      sort: true,
+    },
+    {
+      key: 'height',
+      label: 'Altura',
+      sort: true,
+      actions: {
+        trigger: columnTrigger,
+      },
+    },
+  ];
+
+  const tableWithCellEvents = {
+    config: {
+      data: data,
+      columns: columnsWithCellEvent,
+      pagination: {
+        total: 2,
+        itemsPerPage: 10,
+        page: 1,
+      },
+      loading: false,
+    },
+    events: {
+      emit: events,
+    } as SafeAny,
+  };
+
+  beforeEach(async () => {
+    await sut(tableWithCellEvents);
+  });
+
+  it.each([0, 1])('should render table with clickable cell', async (index) => {
```

Comment:
I don't think you need to iterate this array to validate if your cells are clickable or not.
I would separate this tests in two, one that check if a given cell is clickable and the other one if is not clickable. 

Also, I think it would be more accurate if you check if an event isn't being emmited if you trying to click a cell that don't have the trigger click.